app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
use 
`people: dict[str, Person] = {}`

Reply:
я раніше не зустрічав такого запису, і я не розумію де я його маю використати

Reply:
It is needed to provide type annotation for people attribute. You can change the second line as mentors are targeting it. But don't import annotations, as it will cause tests to fail. Use

people: dict[str, 'Person'] = {}